server/neptune/temporalworker/job/id_generator.go

```diff
@@ -0,0 +1,9 @@
+package job
+
+import "github.com/google/uuid"
+
+type IdGenerator struct{}
+
+func (i *IdGenerator) GenerateID() string {
```

Comment:
I've already added job id support:
https://github.com/lyft/atlantis/blob/25d3a061e24f5658a70889d9eed73d1cb81bbc6b/server/neptune/workflows/internal/sideeffect/uuid.go#L10-L11

you just need to plumb that through